### ISSUE TITLE
[Question] is the lessons page on a course supposed to be accesible on its own?

### ISSUE THREAD

**Issue by Enyeltov** [guillermoscript/lms-front#209]
there is a lessons view separate from the one used in the course page, which can be accesed by the user via clicking lessons in the breadcrumb trail, in case this page is meant to be accesible, it should use the same styling as the lessons on the course page view

Normal lessons view: (Accesible from the course page)
![image](https://github.com/user-attachments/assets/81b7439d-5ce4-400f-86c7-3561603f3c62)

Lessons page view: (accesible from the breadcrumb or directly from the url)
![image](https://github.com/user-attachments/assets/6bb17080-9228-4aea-bd3b-86551607f3df)

**Comment by guillermoscript** [comment]
yeah i think the ui looks cool here, not sure if to change here this @Enyeltov
